### ISSUE TITLE
FID/IS: how can i calculate the conditional models' FID/IS score?

### ISSUE THREAD

**Issue by zyx1213271098** [CompVis/stable-diffusion#421]
I only find the unconditional models fid/is calculate codes in adm_evaluator.py. how can i calculate the conditional models' FID/IS score?

**Comment by KyonP** [comment]
I am also looking for a way to evaluate the FID score. Are there any suggestions or hints?

**Comment by lancel-nv** [comment]
https://github.com/mseitzer/pytorch-fid

**Comment by KyonP** [comment]
> https://github.com/mseitzer/pytorch-fid

Thank you for your reply. I should have elaborated on the question 😅 

SD takes too long to generate all the validation/test sets.

Are there any hints or suggestions? 

Currently, I just sample out a small portion of the test set and benchmark between models.

**Comment by lancel-nv** [comment]
Yeah.. That's True. For an accurate FID value, you might need to use thousands of generated images. I choose to use 2.5K and it needs almost 4 hours in a 4xA100(40G) node... But at least, use as much batch size would help with that.

**Comment by sir-zengqi** [comment]
May I ask which dataset SD uses to measure FID? Can you share the link? I want to know how to reproduce the data in the paper
